tokio/tests/rt_common.rs

```diff
@@ -1017,6 +1017,32 @@ rt_test! {
         });
     }
 
+    #[test]
+    fn coop_unconstrained() {
+        use std::task::Poll::Ready;
+
+        let rt = rt();
+
+        rt.block_on(async {
```

Comment:
This is nit-picky, but I'd like to also see the counter-variant of this test that tests that _without_ `unconstrained`, some of the tasks are _not_ ready. Otherwise we don't know that this test actually tests what we want :)

Reply:
There is in fact one just a few lines above :)

Reply:
You are completely right — I am blind!